### PR TITLE
Build Goptuna Suggestion image in CI

### PR DESCRIPTION
**Pull request by andreyvelich**
Part of https://github.com/kubeflow/katib/issues/1152.

I added Goptuna suggestion build to CI and updated `katib-config` with the new image.

/assign @c-bata 
/cc @johnugeorge @gaocegege

### CODE REVIEW

**Comment by andreyvelich** [comment]
Also, I updated prow.
We should trigger CI if we make changes in `cmd/goptuna`

File: manifests/v1alpha3/katib-controller/katib-config.yaml [hunk 1]
```diff
@@ -46,5 +46,9 @@ data:
             "memory": "200Mi"
           }
         }
+      },
+      "cmaes": {
+        "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-goptuna",
+        "imagePullPolicy": "Always"
```
Comment (c-bata) [inline manifests/v1alpha3/katib-controller/katib-config.yaml:52]: [nits] Most image pull policies of other suggestion service is `IfNotPresent`. So I think you can omit this line.

Comment (andreyvelich) [reply manifests/v1alpha3/katib-controller/katib-config.yaml]: Sure, deleted

**Pull request merged**